contract/AElf.Contracts.Genesis/BasicContractZeroState.cs

```diff
@@ -34,4 +34,6 @@ public partial class BasicContractZeroState : ContractState
     public MappedState<long, ContractCodeHashList> ContractCodeHashListMap { get; set; }
 
     public SingletonState<int> ContractProposalExpirationTimePeriod { get; set; }
+    
+    public MappedState<Address, Address> SignatoryMap { get; set; }
```

Comment:
How about `SignersForAccounts` or `AccountSignerMap`? In our case, it's the physical signer instead of the logical signatory. So Signer is more accurate to the context.